packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -252,3 +255,14 @@ export function initLogger(
 
   return logger;
 }
+
+function getInitialCGC(args: BeaconArgs & GlobalArgs, config: ChainForkConfig, enr: SignableENR): number {
```

Comment:
```suggestion
function getInitialCustodyGroupCount(args: BeaconArgs & GlobalArgs, config: ChainForkConfig, enr: SignableENR): number {
```

would favor avoiding the abbreviation unless we deal with the ENR